### DISCUSSION TITLE
Assigning a key to Refresh (other than f5) doesn't work

### DISCUSSION THREAD

**Discussion by rezelute** [dbeaver/dbeaver#12184] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version Windows 10
- DBeaver version: 7.3.4.202101311042
- Additional extensions: N/A

#### Describe the problem you're observing:
I have tried assigning CTRL+R to Refresh in the key shortcuts then I try to refresh data however it doesnt do anything. Refresh only seems to work when i press F5 or right-click and refresh. I initially thought it might be because of key conflicts however i have unassigned the other shortcut and it still doesn't work. Furthermore, i have tried many other key combinations like ALT+R etc. and they dont work either.
![image](https://user-images.githubusercontent.com/13378850/107345254-4b519b00-6abb-11eb-9358-93eeb036f59c.png)

Is there something I am missing?

Thank you :)

**Comment by uslss** [comment]
thanks for the report

**Comment by ShadelessFox** [comment]
Hello @rezelute,

I cannot reproduce your issue with the same setup (using <kbd>Ctrl</kbd>+<kbd>R</kbd> as a shortcut).

Can you try upgrading and testing against [newest version](https://dbeaver.io/download/)?

**Comment by rezelute** [comment]
Thank you for the reply :). I am running version: 21.0.3.202104181339 and this is what i have in the keys:
![image](https://user-images.githubusercontent.com/13378850/115413418-0ea5cc80-a1ed-11eb-8a2b-3a154446178c.png)

Simple way to replicate:
I edit a field (and dont save), then if i right-click and select Refresh, it gives me the dialog box:
![image](https://user-images.githubusercontent.com/13378850/115413676-46147900-a1ed-11eb-8012-2e9f382eef85.png)

Then i press Cancel (so the field for ex is still highlighted as changed)

If i press CTRL + R, nothing happens.

My point above is that Right click and select Refresh then it is working fine but the shortcut key i am assigning doesnt.

**Comment by ShadelessFox** [comment, marked as answer]
Thanks for the informative response!

I was able to reproduce your issue by having two shortcuts set at the same time. Try clicking <kbd>Unbind Command</kbd> first, then set desired shortcut. I hope that solution suits your needs, because by doing that, well, <kbd>F5</kbd> will not work from this moment.

Let me know if your issue was solved :)

**Comment by rezelute** [comment]
Ah, unbind first seems to have solved it thank you! 😁. Having a single command is perfectly fine thanks.
